components/supervisor/pkg/supervisor/config.go

```diff
@@ -243,8 +243,8 @@ type WorkspaceConfig struct {
 	// WorkspaceClassInfo denotes the detail of workspace class
 	WorkspaceClassInfo *WorkspaceClassInfo `env:"GITPOD_WORKSPACE_CLASS_INFO"`
 
-	// WorkspaceDefaultImage is the default image of current workspace
-	WorkspaceDefaultImage string `env:"GITPOD_WORKSPACE_DEFAULT_IMAGE"`
+	// DefaultWorkspaceImage is the default image of current workspace
+	DefaultWorkspaceImage string `env:"GITPOD_DEFAULT_WORKSPACE_IMAGE"`
```

Comment:
we should have in this file `GetDefaultWorkspaceImage` which will do fallback to workspace-full if env var is empty, clients should not care about it.

Reply:
Would be part of https://github.com/gitpod-io/gitpod/commit/1941376e5d72b8436bf1c6ee91057df1df1b889e